c2corg_ui/static/js/user.js

```diff
@@ -120,4 +120,13 @@ app.UserController.prototype.hasEditRights = function(doctype, options) {
   return this.auth.hasEditRights(doctype, options);
 };
 
+
+/**
+ * @return {boolean}
+ * @export
+ */
+app.UserController.prototype.isModerator = function() {
+  return this.auth.isModerator();
+};
```

Comment:
I don't have a strong opinion but is this alias really needed? You can use ``userCtrl.auth.isModerator()`` in the templates or partials. For instance https://github.com/c2corg/v6_ui/blob/1a6a6f27fcd1ed8b0a5421bcd3e167223112bda7/c2corg_ui/templates/article/edit.html#L82